client/src/actions/userActions.utils.js

```diff
@@ -1,7 +1,12 @@
-import { setChromeStorageLocal, getChromeStorageLocal } from '@cd/services/localStorage';
+import { CONNECTED_ACCOUNT_STORAGE_PATH } from '@cd/constants/settings';
+import { isUsingExtension, setLocalStorageValue, setChromeStorageLocal, getChromeStorageLocal } from '@cd/services/localStorage';
 
 const getConnectedAccountChromeLocalStorage = async () => {
-  return getChromeStorageLocal(["publicKey", "loginOptions"]);
+  try {
+    return getChromeStorageLocal(["publicKey", "loginOptions"]);
+  } catch (err) {
+    return undefined;
```

Comment:
Seem you missed the throw exception case @duc-q .

![Screen Shot 2022-08-14 at 11 43 46](https://user-images.githubusercontent.com/4486806/184522845-37fbb776-ccfc-47ee-821d-2ff7101d7e8d.png)

I will merge this PR first for, we add this case later in another PR. thanks.